minos/common/model/model.py

```diff
@@ -67,6 +72,30 @@ def __init__(self, *args, **kwargs):
         self._fields = dict()
         self._list_fields(*args, **kwargs)
 
+    @classmethod
```

Comment:
The initial approach to integrate `MinosModel` with the logic that serialises and deserialises the data to the `avro` format has been to perform a minimal set of modifications over `MinosAvroValuesDatabase` to be able to work fine with any `data` and `schema` structures.

Probably, a better approach is to perform a code refactor over the `minos.common.protocol` module, but I don't know if fits inside this issue, or a new one could be created specifically for that task.